lib/supply/client.rb

```diff
@@ -292,10 +292,34 @@ def clear_screenshots(image_type: nil, language: nil)
       raise result.error_message if result.error?
     end
 
+    def upload_obb(obb_file_path)
+      result = api_client.execute(
+        api_method: @android_publisher.edits.expansionfiles.upload,
+        parameters: {
+          packageName: current_package_name,
+          editId: current_edit.data.id,
+          expansionFileType: obb_main?(obb_file_path),
+          media: {
+            mimeType: 'application/octet-stream',
+            body: obb_file_path.read
+          }
+        },
+        authorization: auth_client
+      )
+
+      raise result.error_message if result.error?
+    end
+
     private
 
     def ensure_active_edit!
       raise "You need to have an active edit, make sure to call `begin_edit`" unless @current_edit
     end
+
+    def obb_main?(obb_file_path)
```

Comment:
rename to something like 

obb_expansion_file_type

no need of  `?` it's for boolean methods